### DISCUSSION TITLE
Neovim: Can't seem to remap tab to something else

### DISCUSSION THREAD

**Discussion by matheusfillipe** [community/community#20695] [Copilot Conversations] [answered]
I wanted to remap `Tab` to something else on neovim. As it states on the docs should be a matter of doing:

```vim
  imap <silent><script><expr> <C-J> copilot#Accept("")
  let g:copilot_no_tab_map = v:true
```

However that doesn't work and the suggestion stops showing if `g:copilot_no_tab_map` is set to `true`. If I set it to `false` the suggestion shows but the new map doesn't work (tab still works).

**Comment by creasty** [comment, marked as answer]
Make sure the mapping is defined before the initialization of copilot.
Another way is to set `let g:copilot_assume_mapped = v:true` which removes safty check.

https://github.com/github/copilot.vim/blob/c2e75a3a7519c126c6fdb35984976df9ae13f564/autoload/copilot.vim#L401

**Reply by matheusfillipe** [reply]
Thanks, this did it:

```vim
  imap <silent><script><expr> <C-E> copilot#Accept("<End>")
  let g:copilot_no_tab_map = v:true
  let g:copilot_assume_mapped = v:true
  Plug 'github/copilot.vim'
```
